cmake/public/xpu.cmake

```diff
@@ -44,3 +44,13 @@ if(CMAKE_SYSTEM_NAME MATCHES "Linux" AND SYCL_COMPILER_VERSION VERSION_LESS_EQUA
 endif()
 
 string(APPEND XPU_HOST_CXX_FLAGS " -DSYCL_COMPILER_VERSION=${SYCL_COMPILER_VERSION}")
+
+if(DEFINED ENV{XPU_ENABLE_KINETO})
```

Comment:
Seems this new env var will conflict with original `USE_KINETO`, for example, if we set `USE_KINETO=0` but set `XPU_ENABLE_KINETO=1`, what's the behavior here? Suggest to set the new var as second level of original one, that means the new var will effect only under `USE_KINETO=1`. And better to change the new var name to `USE_KINETO_XPU` to align original one. How do you think?

Reply:
`USE_KINETO_XPU` is temporary solution here, because we still not remove level zero header dependency. After we done the dependency remove work, we will remove this temporary code also.